test/e2e_node/system/types_windows.go

```diff
@@ -0,0 +1,52 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import "os/exec"
+
+const DockerEndpoint = "npipe:////./pipe/docker_engine"
```

Comment:
I'm not sure if we're enforcing golint here, but I think so. Then this will fail as it needs a comment